nucleus/common/internal-api/src/main/java/fish/payara/monitoring/collect/MonitoringDataCollection.java

```diff
@@ -108,10 +108,7 @@ private static boolean isGetter(Method method) {
 
     public static void collectRange(MonitoringDataCollector collector, RangeStatistic range) {
         collector
-            .collect(range.getName(), range.getCurrent())
-            .group(range.getName())
-                .collect("LowWaterMark", range.getLowWaterMark())
-                .collect("HighWaterMark", range.getHighWaterMark());
```

Comment:
I was adding low and high watermark from the beginning just because they existed but soon realised that this is the same as all time min/max (of current) already provided for any metric so this just clutters the list of metrics for no good reason wherefore it now got removed again.